cmd/podman/cliconfig/config.go

```diff
@@ -57,6 +57,8 @@ type ImagesValues struct {
 	Noheading bool
 	NoTrunc   bool
 	Quiet     bool
+	RO        bool
```

Comment:
Can we do this as a filter instead? Keeps the frontend cleaner. `--filter readonly=true/false` or similar?

Reply:
Much better idea, thanks.